Process.c

```diff
@@ -472,6 +473,53 @@ void Process_makeCommandStr(Process* this) {
          str = stpcpy(str, SEPARATOR);                                                        \
       } while (0)
 
+   #define CHECK_AND_MARK(str_, prefix_)                                                      \
+      if (String_startsWith(str_, prefix_)) {                                                 \
+         WRITE_HIGHLIGHT(0, strlen(prefix_), CRT_colors[PROCESS_SHADOW], CMDLINE_HIGHLIGHT_FLAG_PREFIXDIR); \
+         break;                                                                               \
+      } else (void)0
+
+   #define CHECK_AND_MARK_DIST_PATH_PREFIXES(str_)                                            \
+      do {                                                                                    \
```

Comment:
@BenBE I think this CHECK_AND_MARK_DIST_PATH_PREFIXES code would be slightly improved by changing the "do {} while (0)" to "if (str[0] == '/')" and removing that check from each independent call site.  That will make sure future changes/callers don't accidentally drop the check, and simplifies each call slightly.

Other than that, this looks fine to me.